libs/blocks/library-config/lists/blocks.js

```diff
@@ -100,10 +100,16 @@ export default async function loadBlocks(blocks, list, query) {
     const doc = parser.parseFromString(html, 'text/html');
     const pageBlocks = doc.body.querySelectorAll('div[class]');
     let matchingBlockFound = false;
+    const classVariationsFound = [];
     pageBlocks.forEach((pageBlock) => {
       if (pageBlock.className === 'section-metadata') {
         return;
       }
+      const blockVariationName = [...pageBlock.classList].join('.');
```

Comment:
I like your approach! 
after the initial classname, the order of the variants doesn't actually matter. 
That means that if there are two same block variants but with a different order of the attributes, they'll both be listed in the library with the current solution. We technically shouldn't be authoring this way, but since we're fixing the duplicate entries, we should probably cover all cases of duplicates. 
You can what I'm talking about with this example. 
https://main--milo--adobecom.hlx.page/drafts/vhargrave/example-variant-ordering

Reply:
```suggestion
      const blockVariationName = [...pageBlock.classList].sort().join('.');
```

Reply:
@vhargrave @chrischrischris Thanks for sharing the feedback. I have updated the statement to sort the classnames before joining them so that the order can be same fro the same classname.
Could you please review and confirm if there are any changes required?